### PR TITLE
A0-367: Fix AlephBFT example

### PR DESCRIPTION
**Pull request by michalseweryn**
An update of libp2p broke our example. This PR fixes that.

### CODE REVIEW

**Review by timorl — DISMISSED** [review]
If it works then LGTM.

File: examples/blockchain/network.rs [hunk 1]
```diff
@@ -372,8 +372,14 @@ impl NetworkManager {
                     }
                 }
                 event = self.swarm.next().fuse() => {
```
Comment (DamianStraszak) [inline examples/blockchain/network.rs:374]: There is also the second example: `dummy_honest.rs` -- if we want to maintain it, then this likely also needs to be changed there.

Another option would be to drop the `dummy_honest` example :)

**Review by timorl — APPROVED** [review]

File: examples/blockchain/network.rs (hunk 1 above)
Comment (timorl) [reply examples/blockchain/network.rs:374]: I think I support dropping it – if we are planning to simplify and improve (i.e. remove libp2p ;P) the main example then it should be enough for any documentation/tutorial purposes.

Comment (michalseweryn) [reply examples/blockchain/network.rs:374]: Fixed :)

Comment (timorl) [reply examples/blockchain/network.rs:374]: I think that is the most disgusting solution you could have picked. xd

Not that I object much.

**Pull request merged**